TelCo.ColorCoder/Test.cs

```diff
@@ -0,0 +1,50 @@
+using System;
+using System.Diagnostics;
+using System.Drawing;
+
+namespace TelCo.ColorCoder
+{
+    /// <summary>
+    /// Test code for the class
+    /// </summary>
+        
+    class Test{
+         
+        /// <param name="args"></param> 
+        private static void Main(string[] args)
+        {
+            TestAllPairs();
+
+            ReferenceManual.PrintMapping(new ConsoleLogger());
+            
+        }
+
+        public static void TestAllPairs()
+        {
+            int actualPairNumber = 1;
+
+            for(int i = 0; i < ColorCoder.colorMapMajor.Length; i++)
```

Comment:
The code to test is parallel to the PrintMapping code. So it's possible that the test code works while the real PrintMapping has an issue. Passing tests would not provide confidence to working functionality.